test/profiles/tizen.profile

```diff
@@ -9,3 +9,8 @@ ENABLE_MODULE_PWM
 ENABLE_MODULE_SPI
 ENABLE_MODULE_TIZEN
 ENABLE_MODULE_UART
+
+# Jerryscript Profile
```

Comment:
IMHO this should be placed into a new file even if this work now.

Reply:
I understand your concern. A new file is added.